docker-entrypoint.sh

```diff
@@ -29,16 +29,21 @@ if [ "$1" = 'postgres' ]; then
 		fi
 		
 		: ${POSTGRES_USER:=postgres}
-		if [ "$POSTGRES_USER" = 'postgres' ]; then
-			op='ALTER'
-		else
-			op='CREATE'
+		: ${POSTGRES_DB:=$POSTGRES_USER}
```

Comment:
I didn't want to change any code for dropping the default `postgres` table but if that would be prefered I could.
I assume I could just add `DROP DATABASE IF EXISTS postgres` to the if statement that creates the POSTGRES_DB